composer.json

```diff
@@ -114,7 +114,8 @@
             "App\\Helpers\\ComposerScripts::preInstall"
         ],
         "post-install-cmd": [
-            "Illuminate\\Foundation\\ComposerScripts::postInstall"
+            "Illuminate\\Foundation\\ComposerScripts::postInstall",
+            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
```

Comment:
The `post-root-package-install` command comes from laravel package itself https://github.com/laravel/laravel/blob/8.x/composer.json. I don't think we should change that.

Reply:
Interesting! It didn't run for me on setup, so something is definitely not working properly, but agreed that it doesn't make sense to depart from the default Laravel setup.

Reply:
It's because `post-root-package-install` would be run installing the package "monica", but not when running `composer install` like "post-install-cmd" command.
I see the interest of that change, but it might have some consequences, and I prefer to do it separately.